Makefile

```diff
@@ -11,7 +11,7 @@ export PATH := $(path_to_add):$(PATH)
 
 GO        := go
 GOBUILD   := CGO_ENABLED=0 $(GO) build $(BUILD_FLAG)
-GOTEST    := CGO_ENABLED=1 $(GO) test -p 3
```

Comment:
The default value equal to the number of CPU cores, @coocood  change it to 3 because the virtual machine for CI is very poor and often OOM at that time.

Reply:
Changed to `-parallel` and gave me the following result:

```
make dev  222.74s user 40.98s system 209% cpu 2:05.77 total
```